test/integration/user_can_view_a_past_order_test.rb

```diff
@@ -0,0 +1,43 @@
+require "test_helper"
+
+class UserCanViewAPastOrder < ActionDispatch::IntegrationTest
+  test 'user can view a past order' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.